src/Compilers/CSharp/Test/Symbol/Compilation/GetSemanticInfoTests.cs

```diff
@@ -2266,16 +2266,8 @@ public static void Main(string[] args)
                 var thread1 = new Thread(() => comp.GlobalNamespace.GetMember<NamedTypeSymbol>("A").GetMembers());
                 var thread2 = new Thread(() => comp.GlobalNamespace.GetMember<NamedTypeSymbol>("IA").GetMembers());
 
-                if (i % 2 == 0)
-                {
-                    thread1.Start();
-                    thread2.Start();
-                }
-                else
-                {
-                    thread1.Start();
-                    thread2.Start();
-                }
+                thread1.Start();
+                thread2.Start();
```

Comment:
The intent here could be to start threads in a different order depending on the condition.